client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-complete/index.tsx

```diff
@@ -63,9 +63,18 @@ const Complete: Step = function Complete( { flow } ) {
 						) }
 						align="center"
 						children={
-							<a href="/domains/manage" className="components-button is-primary manage-all-domains">
-								{ __( 'Manage all domains' ) }
-							</a>
+							<div className="domain-header-buttons">
+								<a href="/setup/domain-transfer/intro" className="components-button is-secondary">
```

Comment:
Maybe we should give it an `onClick` and empty the onboarding store when they click it? Because they'll find the domains they already transferred there.

Reply:
Great Idea Omar, I have addressed it. 🙇